lib/matplotlib/legend.py

```diff
@@ -1107,16 +1117,16 @@ def set_title(self, title, prop=None):
         self.stale = True
 
     def get_title(self):
-        'return Text instance for the legend title'
+        'Return `~.text,.Text` instance for the legend title.'
```

Comment:
pyto

Reply:
Huh?

Reply:
typo :-)
(sorry that joke is getting old)